resizing_hashtable/r_hashtables.py

```diff
@@ -33,7 +39,17 @@ def hash(string, max):
 # Hint: Used the LL to handle collisions
 # '''
 def hash_table_insert(hash_table, key, value):
-    pass
+    index = hash(key, hash_table.capacity)
+    new_pair = LinkedPair(key, value)
+    current_pair = hash_table.storage[index]
+    while current_pair is not None and current_pair.key != key:
+        current_pair = current_pair.next
+    if current_pair is None:
+        new_pair = LinkedPair(key, value)
```

Comment:
A new pair is already created on line 43.

Reply:
I remove it soon I don't remember why I had two.